src/S3/S3Client.php

```diff
@@ -766,25 +767,45 @@ public static function _default_s3_express_identity_provider(array $args)
      * Modifies API definition to remove `Bucket` from request URIs.
```

Comment:
Could you please update the documentation here to include that it also enforces the field `Expires` to be `timestamp` and that adds the synthetic `ExpiresString`  field.

Reply:
done